dvc/repo/fetch.py

```diff
@@ -111,14 +111,7 @@ def cb(result):
                     cb(repo.cloud.pull(used, jobs))
                 except (NoOutputOrStageError, NoRemoteError):
                     pass
-                obj = stage(
-                    odb,
-                    path_info,
-                    repo.repo_fs,
-                    "md5",
-                    jobs=jobs,
-                    follow_subrepos=False,
-                )
+                obj = stage(odb, path_info, repo.repo_fs, "md5", jobs=jobs,)
```

Comment:
There's an interesting question of whether we should apply dvcignore here instead of relying on it being applied in `repo_fs` :thinking: For now we could keep it as is, but it might be a good idea for the future.